lib/eapeak/inject.py

```diff
@@ -399,11 +482,11 @@ def getRSNInformation(self, essid):
 		if rsnInfo is None:
 			rsnInfo = ''  # Did not find rsnInfo in probe response.
 		else:
-			rsnInfo = parseRSNData(rsnInfo.info)
-			rsnInfo = buildRSNData(rsnInfo)
+			rsnInfoDict = parse_rsn_data(rsnInfo.info)
+			rsnInfo = build_rsn_data(rsnInfoDict)
```

Comment:
Could this be condensed as `rsnInfo = build_rsn_data(parse_rsn_data(rsnInfo.info))`? Doesn't look like `rsnInfoDict` is referred to.